### PR TITLE
Added Latest LiveFyre JS Library

### PR DESCRIPTION
**Pull request by itseasy21**
From Community to Enterprise, the **Livefyre Apps Plugin** infuses your website with real-time social content to increase engagement and drive traffic.

### CODE REVIEW

File: files/livefyre/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "LiveFyre"
+github = "https://github.com/itseasy21/jsdelivr"
```
Comment (jimaek) [inline files/livefyre/info.ini]: This needs to be the project's official repo and the fork of jsdelivr

Comment (itseasy21) [inline files/livefyre/info.ini]: But the project is not on github, then how can it be added ?

Comment (jimaek) [inline files/livefyre/info.ini]: Where is it hosted then?

Comment (itseasy21) [inline files/livefyre/info.ini]: I got it from : http://zor.livefyre.com/wjs/v3.0/javascripts/livefyre.js
It is hosted on livefyre servers, which is the comment system that i am using, but they dont have a cdn and neither their JS is available on any public CDN.

Comment (jimaek) [inline files/livefyre/info.ini]: It doesnt look like its an open source project. We dont just host js files. It needs to have a project page and a license.

**Pull request closed without merging**